src/docs/content/tutorials/quickstart/index.md

```diff
@@ -1,213 +1,281 @@
+ - [write your first simulation](#simulation-construction),
+ - [run your simulation locally](#run-the-test-locally), 
+ - [package and upload your simulation to run on Gatling Enterprise Cloud](#package-and-launch-your-test-with-gatling-enterprise-cloud).
 
 {{< alert tip >}}
-Feel free to join our [Gatling Community Forum](https://community.gatling.io) and ask for help **once you've read this documentation**.
+Join the [Gatling Community Forum](https://community.gatling.io) to discuss load testing with other users. Please try to find answers in the documentation before asking for help.
 {{< /alert >}}
 
-### Picking the Right Language
+## Setup
+This section guides you through installation and setting up your developer environment. Gatling has a lot of optionalities, including:
 
-Gatling provides:
-* a Java DSL, introduced in Gatling 3.7, that can also be used in Kotlin
-* the original Scala DSL
+- build tools,
+- CI/CD integrations,
+- Java, Kotlin, and Scala SDKs
 
-When picking a language for using Gatling, we recommend the following rule:
-* if your target Gatling users are Scala developers, use Scala
-* if they are Kotlin developers, use Kotlin
-* otherwise, use Java
+This guide uses Java and the Maven build tool. Gatling recommends that developers use the Java SDK unless they are already experienced with Scala or Kotlin. Java is widely taught in CS courses, requires less CPU for compiling, and is easier to configure in Maven and Gradle. You can adapt the steps to your development environment using reference documentation links provided throughout the guide. 
 
-Java is widely taught in CS courses, requires less CPU for compiling and is easier to configure in maven or gradle.
+### Sign up for Gatling Enterprise Cloud
+Gatling Enterprise Cloud is a fully managed SaaS solution for load testing. Sign up for a [trial account](https://auth.gatling.io/auth/realms/gatling/protocol/openid-connect/registrations?client_id=gatling-enterprise-cloud-public&response_type=code&scope=openid&redirect_uri=https%3A%2F%2Fcloud.gatling.io%2Fr%2Fgatling) to run your first test on Gatling Enterprise Cloud. The [Gatling website](https://gatling.io/features) has a full list of Enterprise features.
 
-### Installing
+### Install Gatling 
+{{< alert prerequisite >}}
+**Prerequisites**  
+Java 11, 17, or 21 64-bit OpenJDK LTS (Long Term Support) version installed on your local machine. We recommend the [Azul JDK](https://www.azul.com/downloads/?package=jdk#zulu).
+{{< /alert >}}
 
-Please check the [installation section]({{< ref "installation" >}}) to pick a setup that matches your needs.
-Non developers are recommended to start with the bundle setup.
-In this tutorial, we will show the commands to use with the bundle setup.
+This guide uses the Maven plugin, which is installed following the [setup instructions]({{< ref "../../reference/current/extensions/maven_plugin#Setup" >}}) in the Documentation. The other available installation methods are as follows: 
```

Comment:
I don't think you should use hard-coded relative links. You should let Hugo compute them or you. @notdryft do you agree?